drivers/hmis_external_apis/spec/factories/external_forms/form_definitions.rb

```diff
@@ -27,5 +27,14 @@
         }
       JSON
     end
+    transient do
+      append_items { nil } # Items to append to FormDefinition content
+    end
+    after(:create) do |instance, evaluator|
+      if evaluator.append_items
+        instance.definition['item'][0]['item'].push(*Array.wrap(evaluator.append_items))
+        instance.save!
+      end
+    end
```

Comment:
This should already be inherited by the `hmis_form_definition` factory, which is listed as its `:parent`. Was that not working?

(Might also make sense to move this factory into `drivers/hmis/spec/factories/hmis/form/definitions.rb`, if that helps)

Reply:
Ah, yeah, thanks for the catch. I think i filched that while this was still in PR https://github.com/greenriver/hmis-warehouse/pull/4649/files but yes, it should be removed.